### ISSUE TITLE
Its giving the below error when starting the application

### ISSUE THREAD

**Issue by hemant-45** [adrianhajdin/project_syncfusion_dashboard#18]
![image](https://user-images.githubusercontent.com/64040781/190873798-b7e75109-19ab-4af1-8f8d-470fa8912308.png)

**Comment by Pannu786** [comment]
Can you share the source code?

**Comment by hemant-45** [comment]
I have downloaded the [project_syncfusion_dashboard] code and then run the npm i command in my system
After that, i have run the npm start command , then this error starts coming

**Comment by Pannu786** [comment]
Try this     **npm install --legacy-peer-deps**. See if this helps.

**Comment by dicucristiann** [comment]
Yes, "npm install" should be the first command you run after cloning a repository.
Because "npm install" is what installs the third-party libraries that your code requires to run. 
Of course, the "package.json" file already exists, but it installs only the missing packages and those that need an update.

**Comment by sinatayebati** [comment]
if you're looking for a much better MERN stack dashboard that's Dockerized and always works, checkout this project:
[admin-dashboard-fullstack](https://github.com/sinatayebati/admin-dashboard-fullstack)
my project is super easy to setup with MIT license + deployment.
